node/src/components/event_stream_server/sse_server.rs

```diff
@@ -174,6 +178,14 @@ impl SseData {
         }
     }
 
+    /// Returns a random `SseData::DeployExpired`
+    pub(super) fn random_deploy_expired(rng: &mut TestRng) -> Self {
+        let deploy = Deploy::random(rng);
```

Comment:
Maybe we can make sure that the example deploy would have indeed expired: `timestamp + ttl < now`.

Reply:
Nice catch ;)
Addressed in 832961befc4a230029dcf15dac84a5407c9433f8